pytorch_lightning/plugins/precision/ipu_precision.py

```diff
@@ -0,0 +1,24 @@
+from typing import Any
+
+from torch import Tensor
+
+from pytorch_lightning.plugins.precision.precision_plugin import PrecisionPlugin
+
+
+class IPUPrecisionPlugin(PrecisionPlugin):
+
+    def __init__(self, precision: int) -> None:
+        super().__init__()
+        self.precision = precision
+
+    def backward(
+        self,
+        closure_loss: Tensor,
+        *args: Any,
+        **kwargs: Any,
+    ) -> Tensor:
+        # IPU internally manages bwd step.
+        return closure_loss
+
+    def clip_gradients(self, *args, **kwargs) -> None:
```

Comment:
`clip_gradients` couldn't be supported?

Reply:
Clipping gradients or norm is not currently supported for IPUs

Reply:
Should we raise an Error or Warning instead?

Reply:
Let's hard crash imo, if someone tries to clip!